From 32ecdb5d33d1faedcc860947ac4a3ec626d593eb Mon Sep 17 00:00:00 2001
From: Darryl Green <darryl.green@arm.com>
Date: Mon, 7 Jan 2019 13:05:37 +0000
Subject: [PATCH 01/43] Add ChangeLog entry

---
 ChangeLog | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 96fa9f334..b3984f145 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -204,10 +204,6 @@ Features
    * Add MBEDTLS_REMOVE_3DES_CIPHERSUITES to allow removing 3DES ciphersuites
      from the default list (enabled by default). See
      https://sweet32.info/SWEET32_CCS16.pdf.
-
-Bugfix
-   * Fix a compilation issue with mbedtls_ecp_restart_ctx not being defined
-     when MBEDTLS_ECP_ALT is defined. Reported by jwhui. Fixes #2242.
    * Run the AD too long test only if MBEDTLS_CCM_ALT is not defined.
      Raised as a comment in #1996.
    * Reduce the stack consumption of mbedtls_mpi_fill_random() which could
-- 
2.17.1

