From 72d7e6cb464c1d1a29aeda2774281cbd589d1d2a Mon Sep 17 00:00:00 2001
From: Sylvain Berfini <sylvain.berfini@belledonne-communications.com>
Date: Mon, 10 Jun 2019 13:52:02 +0200
Subject: [PATCH 36/43] Fixed build issue

---
 library/net_sockets.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/library/net_sockets.c b/library/net_sockets.c
index 5d538bfd5..46ee556d9 100644
--- a/library/net_sockets.c
+++ b/library/net_sockets.c
@@ -323,8 +323,8 @@ int mbedtls_net_accept( mbedtls_net_context *bind_ctx,
     socklen_t n = (socklen_t) sizeof( client_addr );
     socklen_t type_len = (socklen_t) sizeof( type );
 #else
-    int n = (int) sizeof( client_addr );
-    int type_len = (int) sizeof( type );
+    unsigned int n = (unsigned int) sizeof( client_addr );
+    unsigned int type_len = (unsigned int) sizeof( type );
 #endif
 
     /* Is this a TCP or UDP socket? */
-- 
2.17.1

