From 07649eae0e80ace8e016c7cf3cd0ce4cf703a12e Mon Sep 17 00:00:00 2001
From: Peio Rigaux <peio.rigaux@belledonne-communications.com>
Date: Tue, 19 Nov 2019 12:03:31 +0100
Subject: [PATCH 37/43] fixed build error related to net_socket types

---
 library/net_sockets.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/net_sockets.c b/library/net_sockets.c
index 46ee556d9..6285e4370 100644
--- a/library/net_sockets.c
+++ b/library/net_sockets.c
@@ -329,7 +329,7 @@ int mbedtls_net_accept( mbedtls_net_context *bind_ctx,
 
     /* Is this a TCP or UDP socket? */
     if( getsockopt( bind_ctx->fd, SOL_SOCKET, SO_TYPE,
-                    (void *) &type, &type_len ) != 0 ||
+                    (void *) &type, &type_len) != 0 ||
         ( type != SOCK_STREAM && type != SOCK_DGRAM ) )
     {
         return( MBEDTLS_ERR_NET_ACCEPT_FAILED );
-- 
2.17.1

