From 7514fb07bdb32b69bc0534a280dedfe51e212bd4 Mon Sep 17 00:00:00 2001
From: Simon Morlat <simon.morlat@belledonne-communications.com>
Date: Mon, 2 Mar 2020 10:43:28 +0100
Subject: [PATCH 40/43] Use find_program() as a fallback for perl detection.

---
 CMakeLists.txt | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cf630d311..bb8949ee0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,11 +47,17 @@ set(CTR_DRBG_128_BIT_KEY_WARNING "${WARNING_BORDER}"
                          "${WARNING_BORDER}")
 
 find_package(PythonInterp)
-#find_package(Perl)
 
+find_package(Perl)
+if (PERL_EXECUTABLE)
+message(STATUS "Perl was found.")
+else()
+message(WARNING "Perl was not found using the official way, let's try the simple way.")
 find_program(PERL_EXECUTABLE perl PATHS "c:\\MinGW\\msys\\1.0\\bin" )
-if(PERL_EXECUTABLE)
+endif()
 
+
+if(PERL_EXECUTABLE)
     # If 128-bit keys are configured for CTR_DRBG, display an appropriate warning
     execute_process(COMMAND ${PERL_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/config.pl -f ${CMAKE_CURRENT_SOURCE_DIR}/include/mbedtls/config.h get MBEDTLS_CTR_DRBG_USE_128_BIT_KEY
                         RESULT_VARIABLE result)
-- 
2.17.1

