From 160a5178fd9e917ccd46b466cbacf6310f20e0c3 Mon Sep 17 00:00:00 2001
From: ycyang1229 <yc.yang1229@gmail.com>
Date: Fri, 26 Nov 2021 16:16:06 +0800
Subject: [PATCH 4/4] remove nvs operations.

---
 lib/tls/tls.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/tls/tls.c b/lib/tls/tls.c
index 63321c64..bc165b4c 100644
--- a/lib/tls/tls.c
+++ b/lib/tls/tls.c
@@ -142,7 +142,7 @@ lws_tls_server_conn_alpn(struct lws *wsi)
 #endif
 
 #if !defined(LWS_PLAT_OPTEE) && !defined(OPTEE_DEV_KIT)
-#if defined(LWS_PLAT_FREERTOS) && !defined(LWS_AMAZON_RTOS)
+#if 0//defined(LWS_PLAT_FREERTOS) && !defined(LWS_AMAZON_RTOS)
 int alloc_file(struct lws_context *context, const char *filename, uint8_t **buf,
 	       lws_filepos_t *amount)
 {
@@ -217,6 +217,7 @@ int alloc_file(struct lws_context *context, const char *filename, uint8_t **buf,
 	}
 
 	*amount = s;
+	(*buf)[s] = '\0';
 
 bail:
 	if (f)
-- 
2.17.1

