From 138523445d8d1f1d2eff9285094a6c3ce197a9d0 Mon Sep 17 00:00:00 2001
From: ycyang <yc.yang1229@gmail.com>
Date: Tue, 5 Jan 2021 10:21:06 +0800
Subject: [PATCH 4/7] use angle brackets for consistency.

---
 crypto/hash/hmac_mbedtls.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/hash/hmac_mbedtls.c b/crypto/hash/hmac_mbedtls.c
index 78ad71c..c3ee648 100644
--- a/crypto/hash/hmac_mbedtls.c
+++ b/crypto/hash/hmac_mbedtls.c
@@ -48,7 +48,7 @@
 #include "auth.h"
 #include "alloc.h"
 #include "err.h" /* for srtp_debug */
-#include "mbedtls/md.h"
+#include <mbedtls/md.h>
 
 #define SHA1_DIGEST_SIZE 20
 
-- 
2.17.1

