From 7981499af14222003b26897057995437f18ef685 Mon Sep 17 00:00:00 2001
From: ycyang1229 <yc.yang1229@gmail.com>
Date: Thu, 29 Apr 2021 17:25:17 +0800
Subject: [PATCH 2/5] disable ipv6 and program.

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64c0061..bc0efe4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,16 +62,16 @@ if (sctp_inet)
 	add_definitions(-DINET)
 endif ()
 
-option(sctp_inet6 "Support IPv6" 1)
+option(sctp_inet6 "Support IPv6" 0)
 if (sctp_inet6)
 	add_definitions(-DINET6)
 endif ()
 
-option(sctp_werror "Treat warning as error" 1)
+option(sctp_werror "Treat warning as error" 0)
 
 option(sctp_link_programs_static "Link example programs static" 0)
 
-option(sctp_build_programs "Build example programs" 1)
+option(sctp_build_programs "Build example programs" 0)
 
 option(sctp_sanitizer_address "Compile with address sanitizer" 0)
 
-- 
2.17.1

