From 4c78c7a980f0db1da586a30167f08fe23fda9b64 Mon Sep 17 00:00:00 2001
From: ycyang <yc.yang1229@gmail.com>
Date: Tue, 25 Apr 2023 13:45:29 +0800
Subject: [PATCH 7/7] disable the compiling of shared lib.

---
 usrsctplib/CMakeLists.txt | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/usrsctplib/CMakeLists.txt b/usrsctplib/CMakeLists.txt
index 7f45b47..581cf25 100644
--- a/usrsctplib/CMakeLists.txt
+++ b/usrsctplib/CMakeLists.txt
@@ -180,29 +180,29 @@ list(APPEND usrsctp_sources
 	user_socket.c
 )
 
-add_library(usrsctp SHARED ${usrsctp_sources} ${usrsctp_headers})
+#add_library(usrsctp STATIC ${usrsctp_sources} ${usrsctp_headers})
 add_library(usrsctp-static STATIC ${usrsctp_sources} ${usrsctp_headers})
 
-target_include_directories(usrsctp PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+#target_include_directories(usrsctp PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 target_include_directories(usrsctp-static PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
 if(SCTP_USE_MBEDTLS_SHA1)
-	target_include_directories(usrsctp PRIVATE ${MBEDTLS_INCLUDE_DIRS})
+	target_include_directories(usrsctp-static PRIVATE ${MBEDTLS_INCLUDE_DIRS})
 endif()
 
 if (WIN32)
 	message(STATUS "link library: ws2_32")
-	target_link_libraries(usrsctp ws2_32 iphlpapi.lib)
+	#target_link_libraries(usrsctp ws2_32 iphlpapi.lib)
 	target_link_libraries(usrsctp-static ws2_32 iphlpapi.lib)
 endif ()
 
 if(SCTP_USE_MBEDTLS_SHA1)
-	target_link_libraries(usrsctp PRIVATE ${MBEDTLS_LIBRARIES})
+	target_link_libraries(usrsctp-static PRIVATE ${MBEDTLS_LIBRARIES})
 endif()
 
 set_target_properties(usrsctp-static PROPERTIES OUTPUT_NAME "usrsctp")
-set_target_properties(usrsctp PROPERTIES IMPORT_SUFFIX "_import.lib")
-set_target_properties(usrsctp PROPERTIES SOVERSION 1 VERSION 1.0.0)
+#set_target_properties(usrsctp PROPERTIES IMPORT_SUFFIX "_import.lib")
+#set_target_properties(usrsctp PROPERTIES SOVERSION 1 VERSION 1.0.0)
 
 if (NOT DEFINED CMAKE_INSTALL_LIBDIR)
 	SET(CMAKE_INSTALL_LIBDIR lib)
@@ -213,5 +213,5 @@ endif ()
 # INSTALL LIBRARY AND HEADER
 #################################################
 
-install(TARGETS usrsctp usrsctp-static DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS usrsctp-static DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES usrsctp.h DESTINATION include)
-- 
2.34.1

